plugins/GrapesJsBuilderBundle/Assets/library/js/grapesjs-custom.css

```
@@ -361,3 +361,15 @@ html {
     transform: rotate(0deg); }
   100% {
     transform: rotate(360deg); } }
+
+
+/* CKEditor styles */
+.cke-modal .ck-editor .ck-content {
+  min-height: 400px;
+  background: #FFF;
+  color: #000;
```

I'm trying to understand, why are we placing fixed values?

Mautic's CSS supports variables that change based on the theme, this way:

```suggestion
  background: var(--background);
  color: var(--text-primary);
```

If the values are getting replaced by CKEditor's inline styles that reference their own variables, we can place something like this on variables.less (there's a section for CKEditor):

--ckeditor-variable-name: var(--background);
--ckeditor-variable-color: var(--text-primary);

this would link their CSS to ours with 0 effort :)